### PR TITLE
Created Alert Component

### PR DESCRIPTION
**Pull request by nareshpoonia**
Created Alert Component

### CODE REVIEW

**Review by shivampandey0 — APPROVED** [review]
The code looks good. 
Suggestion: include something visual in the description.  Screenshot, video, or gif of the component.

File: styles/alert.css [hunk 1]
```diff
@@ -0,0 +1,109 @@
...
+  .alert-success .fa-check {
+    font-size: 3rem;
+  }
+
+  .alert-warning .msg {
+    font-size: 18px;
+  }
+
+  .alert-success .msg {
+    font-size: 18px;
```
Comment (MaruthiPothuganti) [inline styles/alert.css]: Rems can be used here as it is above 10px

**Review by MaruthiPothuganti — APPROVED** [review]
Suggested a change. Everything else looks good

File: styles/alert.css (hunk 1 above)
Comment (nareshpoonia) [reply styles/alert.css]: Changed unit to rems

**Pull request merged**